tests/providers/amazon/aws/waiters/test_custom_waiters.py

```diff
@@ -424,3 +425,42 @@ def test_steps_failed(self, mock_list_steps):
                 StepIds=[self.STEP_ID1, self.STEP_ID2],
                 WaiterConfig={"Delay": 0.01, "MaxAttempts": 3},
             )
```

Comment:
Not needed for the PR, but I think we're getting to the point where custom waiters have enough traction that these should get split into one-file-per-service like everywhere else...  I don't want you to move the existing tests in this PR but I'll leave it up to you if you want to put it in here or start the process by putting yours in a new file.

Reply:
Ok, I'll put the databrew test in a new file.

Reply:
Thanks.  Maybe this weekend I can make some time to move the others as well.  Shouldn't take too long.